esx_service/utils/auth_api.py

```diff
@@ -239,9 +243,22 @@ def get_default_datastore(name):
         error_info = error_code.generate_error_info(ErrorCode.INTERNAL_ERROR, error_msg)
     return error_info, default_datastore
 
+def is_valid_tenant_name(name):
+    """ Check the given name is a valid tenant name """
+    pattern = "^" + VALID_TENANT_NAME_REG + "$"
+    valid_name_reg = re.compile(pattern)
+    if valid_name_reg.match(name):
```

Comment:
Nit: we can simplify it a little bit by removing the if-else block:

   return valid_name_reg.match(name)